hardware/AMEL/samd/variants/AMEL_SmartEverything/variant.h

```diff
@@ -240,6 +240,11 @@ void setStepUp(uint32_t on);
 */
 bool isOnBattery(void);
 
+
+void gpsForceOn(void);
+void sfxSleep(void);
+void sfxWakeup(void);
+
```

Comment:
i would see these functions more in a dedicated library than in variant code (obfuscated for most of users)